domains/ujjwal.json

```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "ujjwalshriwal",
+    "email": "ujjwalshrivastav16@gmail.com"
+  },
+  "record": {
+    "CNAME": "ujjwalshriwal.github.io"
```

Comment:
404 GitHub Page.

Reply:
Now issue is resolved , you can have a look on GitHub page